Examples/Tests/semi_implicit_electrostatic/inputs_test_2d_semi_implicit_electrostatic_picmi.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
+#
+# --- Test script for the semi-implicit Poisson solver in which a plasma column
+# --- expands inside a conducting cylinder.
```

Comment:
Could you add a bit more text regarding what this test is checking, how to know whether things are behaving as expected, etc.

Reply:
I added more detail in the new test. See description below.